client/jetpack-cloud/sections/agency-dashboard/controller.tsx

```diff
@@ -16,3 +17,10 @@ export function agencyDashboardContext( context: PageJS.Context, next: () => voi
 	);
 	next();
 }
+
+export function pluginManagementContext( context: PageJS.Context, next: () => void ): void {
```

Comment:
You can use the in-built `VoidFunction`
```suggestion
export function pluginManagementContext( context: PageJS.Context, next: VoidFunction ): void {
```

Reply:
Thanks for the review, I have made the changes.